style.css

```diff
@@ -0,0 +1,88 @@
+.clearfix::before,
+.clearfix::after {
+  content: "";
+  clear: both;
+  display: table;
+}
+
+body {
+  margin: 0 auto;
+  font-size: 18px;
+  max-width: 800px;
+  min-width: 670px;
+  color: rgb(126, 126, 126);
+  background-color: rgb(207, 207, 207);
+  padding: 10%;
+}
+
+.border {
+  background-color: rgb(255, 252, 248);
+  padding: 10%;
+  border: 1px transparent;
```

Comment:
Unless the `border-color` is changed at some other point, you can delete this rule. It's not necessary to have a `border` rule in order to use `border-radius` 👍🏻